src/fuzz/build-fuzz.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+export CC=clang
+export CXX=clang++
+export CFLAGS="-g -DFUZZING_BUILD_MODE_UNSAFE_FOR_PRODUCTION -fsanitize=address,undefined -fsanitize=fuzzer-no-link"
+export CXXFLAGS="-g -DFUZZING_BUILD_MODE_UNSAFE_FOR_PRODUCTION -fsanitize=address,undefined -fsanitize=fuzzer-no-link"
+export LIB_FUZZING_ENGINE="-fsanitize=fuzzer"
+
+./configure --no-regex --no-pcre2 --fuzz=$LIB_FUZZING_ENGINE
+make fuzz -j$(nproc)
+
+mkdir build/fuzz_basic_seed
+mkdir build/fuzz_http_parse_seed
+mkdir build/fuzz_json_seed
+
+echo ""
+echo "Run: ./build/\${fuzzer} build/\${fuzzer}_parse_seed src/fuzz/\${fuzzer}_seed_corpus"
```

Comment:
If I want to run the fuzz_basic test, then I do

```
$ ./build/fuzz_basic build/fuzz_basic_seed/ src/fuzz/fuzz_basic_seed_corpus/
```

So it seems the second argument help text should just be

```
build/\${fuzzer}_seed
```

Reply:
Thanks for pointing it out.